engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/management/StandaloneProcessClient.scala

```diff
@@ -68,45 +70,34 @@ class MultiInstanceStandaloneProcessClient(clients: List[StandaloneProcessClient
 
 }
 
-class DispatchStandaloneProcessClient(managementUrl: String, http: Http = Http.default) extends StandaloneProcessClient {
+class DispatchStandaloneProcessClient(managementUrl: String)(implicit backend: SttpBackend[Future, Nothing]) extends StandaloneProcessClient {
 
+  private val managementUri = Uri.parse(managementUrl).get
 
   private implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
-  import pl.touk.nussknacker.engine.dispatch.utils._
-  private val dispatchClient = LoggingDispatchClient(this.getClass.getSimpleName, http)
-
-  import argonaut.ArgonautShapeless._
-  import ProcessName.codec
-  private implicit val stateCodec: CodecJson[RunningState.Value] = Codecs.enumCodec(RunningState)
 
   def deploy(deploymentData: DeploymentData): Future[Unit] = {
-    val deployUrl = dispatch.url(managementUrl) / "deploy"
-    dispatchClient {
-      postJson  (deployUrl, deploymentData) OK asUnit
-    }
+    sttp
+      .post(managementUri.path("deploy"))
+      .body(deploymentData)
+      .send()
+      .map(_ => ())
   }
 
   def cancel(processName: ProcessName): Future[Unit] = {
-    val cancelUrl = dispatch.url(managementUrl) / "cancel" / processName.value
-    dispatchClient {
-      cancelUrl.POST OK asUnit
-    }
+    sttp
+      .post(managementUri.path("cancel", processName.value))
+      .send()
+      .map(_ => ())
   }
 
   def findStatus(name: ProcessName): Future[Option[ProcessState]] = {
-    def notFoundHandler(r: Response): Option[ProcessState] = {
-      if (r.getStatusCode == 404)
-        None
-      else if (r.getStatusCode / 100 == 2)
-        Some(asJson[ProcessState](r))
-      else
-        throw StatusCode(r.getStatusCode)
-    }
-
-    val jobStatusUrl = dispatch.url(managementUrl) / "checkStatus" / name.value
-    dispatchClient {
-      jobStatusUrl > notFoundHandler _
-    }
+    sttp
```

Comment:
There is something like: isSuccess: https://sttp.readthedocs.io/en/latest/responses/basics.html#response-code. I think this is nicer soluton :)